packages/amplify-go-function-runtime-provider/src/runtime.ts

```diff
@@ -161,8 +177,14 @@ export const packageResource = async (request: PackageRequest, context: any): Pr
 };
 
 const winZip = async (src: string, dest: string, print: any) => {
-  // get lambda zip tool
-  await execa(executableName, ['get', '-u', 'github.com/aws/aws-lambda-go/cmd/build-lambda-zip']);
+  // get lambda zip tool with the fix of https://go.dev/doc/go-get-install-deprecation
+  try {
+  // for golang version above > 1.17
+    await execa(executableName, ['install', '-u', 'github.com/aws/aws-lambda-go/cmd/build-lambda-zip']);
```

Comment:
```suggestion
    await execa(executableName, ['install', 'github.com/aws/aws-lambda-go/cmd/build-lambda-zip']);
```

Hey :wave: just a minor tweak, `-u` is not a valid option for `install`